www/source/docs/blueprints.md

```diff
@@ -0,0 +1,66 @@
+# Blueprints
+Blueprints are default implementations of the build phases and runtime hooks for your application. They also provide some default health check hooks where appropriate to ensure your application is functioning reliably. Customized Blueprints can be created to facilitate re-usability of common patterns in your organization for developing, building, and running your applications.
+
+## Getting Started
+### New Projects
+Simply run `hab plan init` to leverage blueprints. Habitat will look in your application code for known project types and the appropriate `pkg_blueprint`.
+
+If you would like to specify the desired blueprint to use, you can run `hab plan init --blueprint core/java8-maven-tomcat`
+
+The resulting plan will contain all the available callbacks and variables relevant to the blueprint type.
+
+### Existing Plans
+
+To begin using Blueprints, you will need to add the appropriate `pkg_blueprints` 
```

Comment:
`pkg_blueprint` - I don't think there will ever be plurality here. The blueprint itself would be composed of others so you wouldn't need to pick more than one